addons/sky_3d/src/Sky3D.gd

```diff
@@ -51,7 +51,7 @@ func set_sky_enabled(value: bool) -> void:
 	sky_enabled = value
 	if not sky:
 		return
-	sky.sky_visible = value
+	#sky.sky_visible = value
```

Comment:
Let's not leave lines randomly commented. Why is this commented out? Should it be removed or reenabled?